pyproject.toml

```diff
@@ -32,6 +35,7 @@ docstring-convention = "google"
 [tool.isort]
 line_length = 99
 profile = "black"
+skip = ["src/legacy.py"]
```

Comment:
Does import sorting need to be disable to `legacy.py`?

Reply:
I don't think so but my though process was if all lints are disabled on this module then controlling imports sorting wouldn't make much of a difference, so I just disabled everything including import sorting checks